script/dump_templates

```diff
@@ -175,6 +184,23 @@ else {
 }
 
 my $client = OpenQA::Client->new(apikey => $options{'apikey'}, apisecret => $options{'apisecret'}, api => $url->host);
+if (1) { # XXX
+    my $group = (keys %{$options{'group'}})[0];
+    $url->path($options{'apibase'} . '/experimental/job_templates_scheduling/' . ($group // ''));
```

Comment:
That ` // ''` might look better on the line above.

Reply:
Except I need it to be undefined for the error message which should say "all groups" when none is specified.